nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -401,6 +401,73 @@ define([
             controlPanel.setExecMessage(Jobs.createCombinedJobState(jobState));
         }
 
+        /* JOB MANAGEMENT */
+
+        /**
+         * Cancel a single job from the batch
+         *
+         * @param {string} jobId
+         */
+        function cancelJob(jobId) {
+            // TODO: implement
+            alert(`Cancel job ${jobId}`);
+            // runtime.bus().emit('request-job-cancellation', {
+            //     jobId: jobId,
+            // });
```

Comment:
this is the correct implementation -- I have left it commented out and will implement with the other actions in another PR.